analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -87,6 +111,47 @@ void Test_NullableValueTypes()
             i.GetHashCode();        // Compliant - safe to call
             i = null;
             i.ToString();           // Compliant - safe to call
+
+            arg.GetType();          // Compliant
+            arg = null;
+            arg.GetType();          // Noncompliant
+            T? localNotNull = new T();
+            localNotNull.GetType(); // Compliant
```

Comment:
Also 
```suggestion
            T? localNotNull = new T();
            localNotNull.GetType(); // Compliant
            T? localNewNull = new T?();
            localNewNull.GetType(); // Noncompliant (this might be a FN at the moment)
            T? localTargetTypeNew = new(); // C#9? or so
            localTargetTypeNew.GetType(); // Compliant localTargetTypeNew is 0
            T? localDefault = default;
            localDefault.GetType(); // Noncompliant (this might be a FN at the moment)
            T? localDefault = default(T);
            localDefaultT.GetType(); // Compliant
```